### PR TITLE
Proposal to fix errors coming from the IPX

### PR DESCRIPTION
**Pull request by guigeek38**
- Ask home assistant to avoid parallel requests because IPX800 doesn't support it (https://developers.home-assistant.io/docs/integration_fetching_data/#request-parallelism)
- Fix debouncing system to avoid Loverace timeout (https://community.home-assistant.io/t/state-not-keeping-updated-on-change/270891)

I was able to test these changes on my setup and everything seems to work. I have no more errors.
What do you think about these changes?

### CODE REVIEW

**Comment by Aohzan** [comment]
Great job, thank you !

**Comment by Aohzan** [comment]
@guigeek38 it seems not working as expected, if I turn off all light, some request are not send to the ipx800

I still get the issue with the state timeout on lovelace by the way

File: custom_components/ipx800/__init__.py [hunk 1]
```diff
@@ -313,7 +315,19 @@ def __init__(self, hass, ipx, update_interval):
...
+        super().__init__(
+            hass,
+            _LOGGER,
+            name=DOMAIN,
+            update_interval=update_interval,
+            request_refresh_debouncer=Debouncer(
+                hass,
+                _LOGGER,
+                cooldown=2,
+                immediate=IpxDataUpdateCoordinator,
```
Comment (Aohzan) [inline custom_components/ipx800/__init__.py:327]: `REQUEST_REFRESH_DEFAULT_IMMEDIATE` instead ?

**Comment by guigeek38** [comment]
> @guigeek38 it seems not working as expected, if I turn off all light, some request are not send to the ipx800

On my setup, I haven't had any errors since I put this fix in place. 
However, my setup uses the **async version** of all calls (before rollbacks), and I applied the fix I suggested for pypx800 in the PR: https://github.com/Aohzan/pypx800/pull/1.
I was able to do network captures (with Wireshark and Tcpdump) and all my requests are sent one after the other to the IPX800.
Do errors occur in deterministic ways on your setup? If so, is it possible to do a network capture to see what's going on?


> I still get the issue with the state timeout on lovelace by the way

I also had some bugs despite this change. I think the right solution as you say is to pass `REQUEST_REFRESH_DEFAULT_IMMEDIATE` to `True`, in order to avoid deboucing...

**Pull request merged**